src/apps/lwaftr/conf.lua

```diff
@@ -19,13 +19,13 @@ local function read_conf(conf_file)
    local input = io.open(conf_file)
    local conf_vars = input:read('*a')
    local full_config = ([[
-      function _conff(policies, C, ipv4, ipv6, ethernet, bt)
+      function _conff(policies, ipv4, ipv6, ethernet, bt)
```

Comment:
This will break existing configs that use C.htonl on VLAN IDs that include the 0x81000 prefix.